.changeset/smooth-pillows-greet.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search': patch
+---
+
+Forward all result set properties to item components, like highlights, for example.
```

Comment:
This is only for the new alpha APIs right? Best be a bit careful and always explicitly mention what parts changes are made to, or I feel there's a risk that readers of the changelog can get confused.

In this case I think we can skip changelog entries altogether though, these APIs aren't released yet anyway.

Reply:
Yes, this is only for the new alpha APIs...
That makes sense, I'll remove the changeset entries, thanks for pointing it out 🙌🏻

Reply:
Done: https://github.com/backstage/backstage/pull/20265/commits/1b51bf1f157cb75c6684f96f4dfebb83c7f55f9c